app/style/app.css

```diff
@@ -1,6 +1,6 @@
 body {
   margin: 0;
-  font-size: 10px;
+  font-size: 20px;
```

Comment:
This affects the text inside the alert dialog. It's almost unreadable without this change.